src/index.js

```diff
@@ -60,7 +60,7 @@ class ProcessHtml {
         const ignoredFromPartial = ignoreLinksFromPartialMatches.filter(partial => href.indexOf(partial) >= 0);
 
         if (ignoreLinks.indexOf(href) < 0 && ignoredFromPartial.length === 0) {
-          source += `\nimport '${path}';\n`;
+          source += `\nimport '${path.replace(/\\/g, '\\\\')}';\n`;
```

Comment:
hmm... 🤔 kk I see normally this is what most loaders use to escape their contents, but seems to not apply here :)